examples/parlai_chat_task_demo/conf/custom_simple.yaml

```diff
@@ -6,8 +6,8 @@ mephisto:
     task_name: parlai-chat-example
     task_title: "Test ParlAI Simply Built Chat Task"
     task_description: >
-      This is a simple chat between two people 
-      used to demonstrate the functionalities around using Mephisto 
```

Comment:
This was a good bug that was caught. The trailing whitespace here was causing double spaces to be added to the string.